frontend/components/ui/blocknote/editor/slash-menu-config.ts

```
@@ -4,6 +4,7 @@ import { createElement } from "react";
 import type { ReactElement } from "react";
 import type { CustomBlockNoteEditor } from "@/lib/blocknote/schema";
 import type { CalloutType } from "@/lib/blocknote/types";
+import { Code, FileCode, Terminal } from "lucide-react";
```

ESLint

[error] 'FileCode' is defined but never used.

(@typescript-eslint/no-unused-vars)